diff -urN aa/libbonoboui-2.8.1/tools/browser/bonobo-browser.desktop.in libbonoboui-2.8.1/tools/browser/bonobo-browser.desktop.in
--- aa/libbonoboui-2.8.1/tools/browser/bonobo-browser.desktop.in	2004-09-24 01:21:46.000000000 +0200
+++ libbonoboui-2.8.1/tools/browser/bonobo-browser.desktop.in	2005-02-05 17:24:46.820752208 +0100
@@ -1,8 +1,9 @@
 [Desktop Entry]
+Encoding=UTF-8
 _Name=Bonobo Component Browser
 _Comment=Shows available Bonobo components
 Exec=bonobo-browser
-Terminal=0
+Terminal=false
 Type=Application
 Icon=
-Categories=Application;System;
+Categories=System;
