diff -aurN libbonoboui-2.14.0.orig/tools/browser/bonobo-browser.desktop.in libbonoboui-2.14.0/tools/browser/bonobo-browser.desktop.in
--- libbonoboui-2.14.0.orig/tools/browser/bonobo-browser.desktop.in	2006-02-24 14:40:48.000000000 +0100
+++ libbonoboui-2.14.0/tools/browser/bonobo-browser.desktop.in	2006-03-14 23:12:40.403699536 +0100
@@ -5,7 +5,7 @@
 Exec=bonobo-browser
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Application;System;
+Categories=GNOME;GTK;System;
 NoDisplay=true
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=bonobo
