diff -urN libbonoboui-2.6.1.org/configure.in libbonoboui-2.6.1/configure.in
--- libbonoboui-2.6.1.org/configure.in	2004-05-14 23:08:11.165376312 +0200
+++ libbonoboui-2.6.1/configure.in	2004-05-14 23:08:24.340373408 +0200
@@ -27,7 +27,7 @@
 gnomelocaledir='${prefix}/${DATADIRNAME}/locale'
 AC_SUBST(gnomelocaledir)
 
-ALL_LINGUAS="af am ar az be bg bn ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gu he hi hr hu id is it ja kn ko li lt lv ml mn mr ms ne nl nn no pa pl pt pt_BR ro ru sk sl sq sr sr@ije sr@Latn sv ta tr uk vi wa yi zh_CN zh_TW"
+ALL_LINGUAS="af am ar az be bg bn ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gu he hi hr hu id is it ja kn ko li lt lv ml mn mr ms nb ne nl nn pa pl pt pt_BR ro ru sk sl sq sr sr@ije sr@Latn sv ta tr uk vi wa yi zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 AC_SUBST(CFLAGS)
